schedulers/nbt/lib/scheduler_nbt.cc

```diff
@@ -36,6 +36,10 @@ void scheduler_nbt::initialize(flat_graph_sptr fg, runtime_monitor_sptr fgmon)
     //  By default, one Thread Per Block
 
     auto blocks = fg->calc_used_blocks();
+    for (auto& b : blocks)
+    {
```

Comment:
There are some style inconsistencies in this code. Like the linebreak before `{`, and in other places the member variables that begin with underscore (not `d_`).

Are these intentional?

Reply:
All style guides are wrong, so all I ask for is consistency. :-)

As long as it's solved before "going live".